.github/workflows/cloud_csharp_client_test.yaml

```diff
@@ -0,0 +1,100 @@
+name: Hazelcast cloud test with csharp client
+
+on:
+  workflow_dispatch:
+    inputs:
+      organization_name:
+        description: Default is hazelcast, but if you would like to run the workflow with your forked repo, set your github username
+        required: true
+        default: hazelcast
+      branch_name:
+        description: Name of the branch to test client from
+        required: true
+        default: master
+      base_url:
+        description: Base url of the cloud env. I.e https://uat.hazelcast.cloud
+        required: true
+        default: https://uat.hazelcast.cloud
+      hzVersion:
+        description: Version of hazelcast (For cloud standard cluster, it has different versions, i.e 5.0.2-2)
+        required: true
+
+jobs:
+  test_standard_cluster:
+    runs-on: ubuntu-latest
+    name: Hazelcast cloud standard cluster tests with csharp
+    strategy: 
+      matrix: 
+        fwks: [netcoreapp3.1,net5.0,net6.0]
```

Comment:
Do we compile things differently depending on different .net versions? Testing the client with different frameworks is good, but I remember these tests were pretty long-running. I would like to avoid it if possible

Reply:
Actually, that was the approach. Cloud tests are done in one version which is latest one mostly. So, you may right. It's enough to test latest one. I think I concerned too much :/ 

I'll update it to test with latest version.